training-react/tic-tac-toe/src/app/screens/Home/screens/Profile/components/Form/index.js

```diff
@@ -23,19 +23,19 @@ function Form({ handleSubmit, msgUpdate, hasError }) {
           update
         </button>
       </form>
-      {!hasError ? (
-        <span className={styles.successUpdate}>{msgUpdate}</span>
-      ) : (
-        <span className={styles.failureUpdate}>{msgUpdate}</span>
-      )}
+      {isSuccessUpdate ? (
+        <span className={styles.successUpdate}>Success update user data</span>
+      ) : hasError ? (
```

Comment:
You could replace this for: `hasError && <span className={styles.failureUpdate}>Error in update user data</span>`